src/Test.FunctionalTests.BinSkim.Driver/BaselineTests.cs

```diff
@@ -429,6 +429,10 @@ private SarifLog RunRules(StringBuilder sb, string inputFileName, Sarif.SarifVer
 
             actualText = actualText.Replace(@"""Sarif""", @"""BinSkim""");
             actualText = actualText.Replace(@"        ""fileVersion"": ""15.0.0""," + Environment.NewLine, string.Empty);
+            actualText = actualText.Replace(@"        ""fileVersion"": ""15.0.0""," + Environment.NewLine, string.Empty);
+
+            actualText = Regex.Replace(actualText, @"\s*""product""[^\n]+?\n", Environment.NewLine);
```

Comment:
>actualText [](http://example.com/codeflow?start=39&length=10)

This change eliminates diff noise in testing when there are actual meaningful diffs (in change of analysis output). There is also a discrepancy in a default property bag item, which I think comes from the SARIF SDK so pushing out updating this for now.